hedyorg/hedy
# [BUG] fix public adventure page

**Description**
The adventures table requires the `public` field to be a boolean. This fixes the issue.

**How to test**
- go to the `for-teachers` page
- click on  `public adventures`
- everything should work; also on alpha!
